cloudify_rest_client/client.py

```diff
@@ -267,6 +273,7 @@ def close(self):
 
 class CloudifyClient(object):
     """Cloudify's management client."""
+    client_class = HTTPClient
```

Comment:
the cli uses a different client when connecting to a cluster (one that stores the active manager in profile etc)

the agent will probably use another implementation for the cluster too